aten/src/ATen/native/sparse/SparseUtils.h

```diff
@@ -42,20 +42,29 @@ inline bool _check_device(ArrayRef<Tensor> ts) {
   return true;
 }
 
-inline void _raw_resize_sparse(const SparseTensor& self, int64_t sparseDims, int64_t denseDims, IntList size) {
+// TODO: remove this function when is USE_TH_SIZE_ZERO_DIM enabled by default
+inline void _raw_resize_sparse_legacy(const SparseTensor& self, int64_t sparseDims, int64_t denseDims, IntList size) {
+#ifndef USE_TH_SIZE_ZERO_DIM
   _get_sparse_impl(self)->raw_resize_(sparseDims, denseDims, size);
```

Comment:
why is just the implementation ifdefed?

Reply:
This is actually because `_raw_resize_sparse_legacy` (this function) is called from `sparse_raw_resize_legacy_`, which is defined in `native_functions.yaml` and can't be gated behind the `USE_TH_SIZE_ZERO_DIM` flag.

Reply:
Fixed